### PR TITLE
add store gateway overview to instance overview dashboard

### PR DESCRIPTION
**Pull request by coleenquadros**
https://issues.redhat.com/browse/RHOBS-705

### CODE REVIEW

**Comment by coleenquadros** [comment]
test dashboard - https://grafana.stage.devshift.net/d/Xt6-Qb9Vk/rhobs-instance-utilization-overview-test-2?orgId=1&refresh=10s&var-datasource=telemeter-prod-01-prometheus&var-namespace=observatorium-metrics-production&var-job=observatorium-thanos-store-shard-2&var-interval=5m

File: observability/dashboards/rhobs-instance-utilization-overview.libsonnet [hunk 1]
```diff
@@ -1,20 +1,20 @@
 local config = (import '../config.libsonnet').thanos;
 local utils = import 'github.com/thanos-io/thanos/mixin/lib/utils.libsonnet';
 local g = import 'github.com/thanos-io/thanos/mixin/lib/thanos-grafana-builder/builder.libsonnet';
-local template = import 'grafonnet/template.libsonnet';
+local grafana = import 'grafonnet/grafana.libsonnet';
```
Comment (douglascamata) [inline observability/dashboards/rhobs-instance-utilization-overview.libsonnet:4]: Curious about the reason for this change. We only use `grafana.template`. Why not to keep the shorter version? 🤔

**Review by douglascamata — APPROVED** [review]
Awesome! Left a little curious question, but not a blocker at all.

The Store GW dashboard row doesn't have the layout issue. How did you fix it, @coleenquadros? Can't wait to apply the fix to all rows.

File: observability/dashboards/rhobs-instance-utilization-overview.libsonnet (hunk 1 above)
Comment (coleenquadros) [reply observability/dashboards/rhobs-instance-utilization-overview.libsonnet]: this was a part of the refactoring. Ill remove it for now

**Comment by coleenquadros** [comment]
> Awesome! Left a little curious question, but not a blocker at all.
> 
> The Store GW dashboard row doesn't have the layout issue. How did you fix it, @coleenquadros? Can't wait to apply the fix to all rows.

I am trying to figure that out too. I didnt do anything apart from picking the panels from the mixin 😕

**Pull request merged**